tiledb/api/c_api/subarray/subarray_api.cc

```diff
@@ -97,6 +97,19 @@ capi_return_t tiledb_subarray_add_point_ranges(
   return TILEDB_OK;
 }
 
+capi_return_t tiledb_subarray_add_point_ranges_var(
+    tiledb_subarray_t* subarray,
+    uint32_t dim_idx,
+    const void* buffer_val,
+    uint64_t buffer_val_size,
+    const uint64_t* buffer_off,
+    uint64_t buffer_off_size) {
```

Comment:
The signature is quite different from the `tiledb_subarray_add_point_ranges` one. Could we follow the same pattern and logic as `tiledb_subarray_add_range_var` and have a signature for this one of that form instead:
```
capi_return_t tiledb_subarray_add_point_ranges(
    tiledb_subarray_t* subarray,
    uint32_t dim_idx,
    const void* start,
    uint64_t start_size,
    const void* end,
    uint64_t end_size)
 ```

Reply:
I get it now, there's no `end` as those are point ranges. However, I'd rename the arguments to something like:
```
capi_return_t tiledb_subarray_add_point_ranges(
    tiledb_subarray_t* subarray,
    uint32_t dim_idx,
    const void* start,
    uint64_t start_size,
    const void* start_offsets,
    uint64_t start_offsets_size)
```